.github/workflows/tests-rs-sdk-grpc-coverage.yml

```
@@ -0,0 +1,123 @@
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - 'packages/rs-sdk/**'
+      - 'packages/dapi-grpc/protos/platform/v0/platform.proto'
+      - '.github/workflows/tests-rs-sdk-grpc-coverage.yml'
+      - '.github/scripts/check-grpc-coverage.py'
+      - '.github/grpc-queries-cache.json'
+  push:
+    branches:
+      - master
+      - 'v[0-9]+\.[0-9]+-dev'
```

actionlint (1.7.7)

character '\' is invalid for branch and tag names. only special characters [, ?, +, *, \, ! can be escaped with \. see `man git-check-ref-format` for more details. note that regular expression is unavailable. note: filter pattern syntax is explained at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

(glob)